src/features/MovieList/MovieTile/styled.js

```diff
@@ -1,125 +1,158 @@
         font-size: 16px;
+        margin: 0 0 4px;
+    }
+
+    @media (max-width: ${({ theme }) => theme.breakpoint.mobile}px){
+        font-size: 4vw;
     }
 `;
 
-export const Year = styled.p`
+export const Year = styled.span`
     margin: 0;
     font-size: 16px;
     line-height: 150%;
     color: ${({ theme }) => theme.color.darkerGray};
 
-    @media (max-width: ${({ theme }) => theme.breakpoint.mobile}px) {
+    @media (max-width: ${({ theme }) => theme.breakpoint.tablet}px){
         font-size: 13px;
-        line-height: 130%;
+    }
+
+    @media (max-width: ${({ theme }) => theme.breakpoint.mobile}px){
+        font-size: 2.5vw;
     }
 `;
 
-export const TagsContainer = styled.div`
-    margin-top: 8px;
+export const TagsContainer = styled.ul`
+    margin: 8px 0 0;
+    padding: 0;
+    gap: 8px;
     display: flex;
     flex-wrap: wrap;
-    gap: 8px;
+    list-style: none;
+    margin-bottom: 0;
 `;
 
-export const Tag = styled.p`
-    margin: 0px;
+export const Tag = styled.li`
+    margin: 0;
     padding: 8px 16px;
     background-color: ${({ theme }) => theme.color.gray};
     border-radius: 5px;
     font-size: 14px;
-    @media (max-width: ${({ theme }) => theme.breakpoint.mobile}px) {
+    
+    @media (max-width: ${({ theme }) => theme.breakpoint.tablet}px){
         padding: 4px 8px;
         font-size: 10px;
     }
+
+    @media (max-width: ${({ theme }) => theme.breakpoint.mobile}px){
+        padding: 1vw 2vw;
+        font-size: 2vw;
+    }
 `;
 
 export const RatingContainer = styled.div`
-    margin-top: 39px;
+    margin-top: 12px;
     display: flex;
+    flex-wrap: wrap;
     align-items: center;
     align-content: flex-end;
     gap: 12px;
-
-    @media (max-width: ${({ theme }) => theme.breakpoint.mobile}px) {
+    
+    @media (max-width: ${({ theme }) => theme.breakpoint.tablet}px){
         gap: 8px;
-        margin-top: 12px;
+    }
+
+    @media (min-width: ${({ theme }) => theme.breakpoint.tablet}px){
+        flex: 1;
     }
 `;
 
 export const StyledStar = styled(Star)`
-    @media (max-width: ${({ theme }) => theme.breakpoint.mobile}px) {
+    width: 24px;
+    height: 24px;
+    fill: none;
+
+    @media (max-width: ${({ theme }) => theme.breakpoint.tablet}px){
         width: 16px;
+        height: 16px;
+    }
+
+    @media (max-width: ${({ theme }) => theme.breakpoint.mobile}px){
+        width: 3vw;
+        height: 3vw;
     }
 `;
 
-export const Rate = styled.p`
-    margin: 0px;
+export const Rate = styled.span`
     font-weight: 600;
     font-size: 16px;
 
-    @media (max-width: ${({ theme }) => theme.breakpoint.mobile}px) {
+    @media (max-width: ${({ theme }) => theme.breakpoint.tablet}px){
         font-size: 13px;
     }
+
+    @media (max-width: ${({ theme }) => theme.breakpoint.mobile}px){
+        font-size: 3vw;
+    }
 `;
 
-export const Votes = styled.p`
-    margin: 0px;
+export const Votes = styled.span`
     font-weight: 400;
+    font-size: 16px;
```

Comment:
`font-weight: 400;` and `font-size: 16px;` are default `<span>` values.

Reply:
fixed